### PR TITLE
Include a prestart script to fix socket permissions

### PR DESCRIPTION
**Pull request by dprotaso**
The vcap user doesn't have permission to write to the docker socket
hence we make it read/writeable by all

### CODE REVIEW

File: tests/run-socket.sh [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+
+cpi_path=$PWD/cpi
+
+rm -f creds.yml
+
+echo "-----> `date`: Create dev release"
+bosh create-release --force --dir ./../ --tarball $cpi_path
+
+echo "-----> `date`: Create env"
+bosh create-env ~/workspace/bosh-deployment/bosh.yml \
```
Comment (cppforlife) [inline tests/run-socket.sh]: where did you end up running this against? cppforlife/docker-deployment?

**Comment by dprotaso** [comment]
docker for mac

**Comment by dsboulder** [comment]
Any chance you'll accept this DK?

**Comment by dprotaso** [comment]
Closing due to no action

**Pull request closed without merging**